hicommonwealth/commonwealth
# Fix new account onboard flow breaking mid way in communities

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/3913

## Description of Changes
The app uses emitters to trigger app level re-renders, one such re-render was triggered midway in account creation flow in communities. This pr adds a fix to only trigger that re-render when the account has been fully onboarded.

## Test Plan
- Create a new account outside community (it should have correct account creation flow)
- Login with an existing account outside community (it should behave normally)
- Create a new account inside community (it should have correct account creation flow)
- Login with an existing account inside community (it should behave normally)

## Deployment Plan
N/A

## Other Considerations
N/A
